pyvrp/tests/test_ProblemData.py

```diff
@@ -185,11 +185,14 @@ def test_matrix_access():
     data = ProblemData(
         clients=clients,
         vehicle_types=[VehicleType(1, 2)],
-        distance_matrix=dist_mat,  # type: ignore
-        duration_matrix=dur_mat,  # type: ignore
+        distance_matrix=dist_mat,
+        duration_matrix=dur_mat,
     )
 
+    assert_allclose(data.distance_matrix(), dist_mat)
+    assert_allclose(data.duration_matrix(), dur_mat)
```

Comment:
I've added this (back in, it was there before we removed matrix access) to make sure getting the matrix stuff from C++ also works fine.